common/src/main/resources/assets/voicechat/lang/en_us.json

```diff
@@ -1,217 +1,212 @@
 {
   "key.categories.voicechat": "Voice Chat",
-  "key.push_to_talk": "Push To Talk",
+  "key.push_to_talk": "Push to Talk",
   "key.whisper": "Whisper",
   "key.mute_microphone": "Mute Microphone",
-  "key.disable_voice_chat": "Disable Voice Chat",
+  "key.disable_voice_chat": "Disable Voice Chat Sounds",
   "key.hide_icons": "Hide Voice Chat Icons",
   "key.voice_chat_settings": "Voice Chat Settings",
-  "key.voice_chat": "Voice Chat GUI",
-  "key.voice_chat_group": "Voice Chat Group",
+  "key.voice_chat": "Voice Chat Menu",
+  "key.voice_chat_group": "Group Management",
   "key.voice_chat_toggle_recording": "Toggle Recording",
   "key.voice_chat_adjust_volumes": "Adjust Volumes",
   "gui.voicechat.voice_chat.title": "Voice Chat",
   "gui.voicechat.voice_chat_settings.title": "Voice Chat Settings",
   "gui.voicechat.adjust_volume.title": "Adjust Volumes",
-  "gui.voicechat.select_microphone.title": "Select Microphone",
-  "gui.voicechat.select_speaker.title": "Select Speaker",
-  "gui.voicechat.group.title": "Group Chat",
-  "gui.voicechat.join_create_group.title": "Join or Create Group",
+  "gui.voicechat.select_microphone.title": "Select a Microphone",
+  "gui.voicechat.select_speaker.title": "Select Speakers",
```

Comment:
It all comes down to what someone thinks of when they hear "speaker" and what they think of when they hear "speakers" instead.

Typical computer and laptop setups usually have either two speakers (left and right), headphones (which are technically also a pair of speakers), or both. A single built-in speaker is now rather rare.

From my perspective, it appears to me that when you think of speakers, you usually think of a pair of them, so "speakers" seemed to me like a natural choice here.

However, you rightly pointed out that the word "speakers" may suggest that the user is allowed to choose multiple pairs of them for the audio output. On the other hand, the word "speaker" may suggest that the user has to decide which one and only one of them should be used at a time — left, right, center, or maybe the rear one.

Obviously, all of this becomes clear once you click and make your choice, so I'll leave it up to you to decide what to do in this case. I'm waiting for your response on this matter.

Reply:
> However, you rightly pointed out that the word "speakers" may suggest that the user is allowed to choose multiple pairs of them for the audio output.

I never had a user thinking that or complaining about the wording, though I could think that people would be confused about the plural.
Also for consistency it would be better IMO to have
```
Select a Microphone
Select a Speaker
```

Reply:
That's what I wanted to hear. Thank you. In that case, I'll restore the previous version in a moment.